hack/pr-infra-deployments.sh

```diff
@@ -23,10 +23,13 @@ set -o pipefail
 set -o nounset
 
 RELEASE_BUNDLE_TAG="git-$(git rev-parse --short HEAD)"
-RELEASE_BUNDLE_TAG="git-d47b70d"
 RELEASE_BUNDLE_DIGEST="$(skopeo manifest-digest <(skopeo inspect --raw "docker://quay.io/hacbs-contract/ec-release-policy:${RELEASE_BUNDLE_TAG}"))"
 RELEASE_BUNDLE_REF="quay.io/hacbs-contract/ec-release-policy:${RELEASE_BUNDLE_TAG}@${RELEASE_BUNDLE_DIGEST}"
 
+DATA_BUNDLE_TAG="git-$(git rev-parse --short HEAD)"
+DATA_BUNDLE_DIGEST="$(skopeo manifest-digest <(skopeo inspect --raw "docker://quay.io/hacbs-contract/ec-policy-data:${RELEASE_BUNDLE_TAG}"))"
```

Comment:
`RELEASE_BUNDLE_TAG` is used instead of `DATA_BUNDLE_TAG`. Would it make sense to just have a `BUNDLE_TAG` for both?

Reply:
yeah probably.

Reply:
Done.